lib/src/config/config.rs

```
@@ -814,7 +764,7 @@ impl ClientConfigBuilder {
                 let bind_to = rpc_config
                     .bind
                     .as_ref()
-                    .and_then(|addr| addr.into_ip_address());
+                    .map(|addr| addr.parse().expect(" Error parsing the RPC server address"));
```

I'd prefer to return a graceful error